### ISSUE TITLE
Hyperparameter tuning in the model for better performance by introduction of callback function.

### ISSUE THREAD

**Issue by Rishi2003Das** [camicroscope/tfjs-models#6]
**Describe the bug**
Finetuning the model of Lymphoma Cancer Classification and introducing a callback fuction to get the best epoch.
**To Reproduce**
1. Go to Breadcrumbs[tfjs-models](https://github.com/camicroscope/tfjs-models/tree/master)/[Classification Sample Models](https://github.com/camicroscope/tfjs-models/tree/master/Classification%20Sample%20Models)/[lymphoma-cancer-classification](https://github.com/camicroscope/tfjs-models/tree/master/Classification%20Sample%20Models/lymphoma-cancer-classification)
/model.ipynb

**Expected behavior**
Better learning rate with a callback function to get the best epoch and thus best result.

Thank you
